docs/pages/usage/verifying.md

```diff
@@ -0,0 +1,79 @@
+# Verifying Stone Proof Using the Binary
+
+This guide will walk you through how to verify a Stone proof using the provided binaries. It includes steps for both setting up the environment using Docker and manually installing the required dependencies.
+
+- Ensure Docker is installed on your machine. You can refer to [Docker’s official documentation](https://docs.docker.com/get-docker/) for installation instructions.
+
+## Creating and Verifying a Test Proof Using Binaries
+
+- Clone the repository:
+
+```bash
+git clone https://github.com/dipdup-io/stone-packaging.git /tmp/stone-packaging
+```
+
+- Navigate to the example test directory
+
+```bash
+cd /tmp/stone-packaging/test_files/
+```
+
+- Download the Binary Files:
+
+Copy or download the binary files from the latest release to this directory.
```

Comment:
Please, refer to the Install section here. Use relative link to https://github.com/dipdup-io/stone-packaging/blob/master/docs/pages/install/binaries.md